Description: Add the new command to the desktop icon
Author: Dag Heyman <dag@yubico.com>
Forwarded: yes

--- a/resources/yubioath.desktop
+++ b/resources/yubioath.desktop
@@ -2,7 +2,7 @@
 Name=Yubico Authenticator
 GenericName=Yubico Authenticator
 Comment=Graphical interface for displaying OATH codes with a Yubikey
-Exec=yubioath
+Exec=yubioath-gui
 Icon=yubioath
 StartupNotify=false
 Terminal=false
